index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Alison Zerbe</title>
+        <link href="styles/style.css" rel="stylesheet">
+        <!-- icon by AlfredoCreates.com -->
+        <!-- where do you put this attribution -->
+    </head>
+    <header>
+        <nav>
+            <!-- <img src="assets/triangle.png"> -->
+            <!-- <img src="assets/pyramid.svg" /> -->
+            <p>Alison Zerbe</p>
+            <ul>
+                <li><a href=about.html>About Me</a></li>
+                <li><a href="portfolio.html">Projects</a></li>
+                <li><a href="code-journal.html">Life At Ada</a></li>
+            </ul>
+        </nav>
+    </header>
+    <body>
+        <section id="banner">
+            <h1>Hello! I'm Alison!</p>
+            <h3>I am a... </h3>
+            <h3>in-progress software engineer, education addict,
+                language activist, globetrotter</h3>
+            <img src="assets/computer.svg" />
+            <img src="assets/education.svg" />
+            <img src="assets/thought_bubbles.svg" />
+            <img src="assets/globetrotter.svg" />
+        </section>
+
+        <section id="image">
+            <img src="https://unsplash.it/400">
+        </section>
+
```

Comment:
Image should have alt text. Also I am really not sure I would put one single image inside a `<section>` tag. I'd probably go with a `<div>` for that.